dinput_hook/renderer_hook.cpp

```diff
@@ -783,6 +873,10 @@ void imgui_Update() {
             std::abort();
 
         fprintf(hook_log, "[OGL_imgui_Update] imgui initialized.\n");
+
+        // Error callback
+        glEnable(GL_DEBUG_OUTPUT_SYNCHRONOUS);
```

Comment:
i think this initialization and the message callback should be in Window.c, or is there any reason they are here?